Extension/webpack.config.js

```diff
@@ -54,14 +54,4 @@ const config = {
     }
 }
 
-if (process.argv.includes('--vscode-nls')) {
```

Comment:
Isn't this removing the nls rewriting, which is required for localization?

Reply:
Was there a reason this was removed?  Does it cause a failure now?  Localization of strings (the ones implemented with localize() in TypeScript code) cannot work when webpacked without this.

Reply:
It doesn't work. I get some error like "webpack doesn't support --vscode-nls". How do I test the localization case? There might be some new way to get it to work.

Reply:
I'm seeing translated strings in most cases, but not in the Output window.

Reply:
Yeah, Output window and Find All References "type" are not translated.

Reply:
The error is
```
[webpack-cli] Error: Unknown option '--vscode-nls'
[webpack-cli] Run 'webpack --help' to see available commands and options
```
I didn't see what change was causing this in the migration docs at https://webpack.js.org/blog/2020-10-10-webpack-5-release/ , but it sounds like it's handling command line arguments different.

Do you have any idea what to change to make it work?

Reply:
We had to do something similar in CMake Tools.  It looks like you landed on about the same approach.
[PR](https://github.com/microsoft/vscode-cmake-tools/pull/1475/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json)

Reply:
I've verified the loc stuff works.